File: Core/Entities/AppDetails.cs
namespace Core.Entities
{
    public class AppDetails : BaseEntity
    {
        public string CompanyName { get; set; }
        public string MainLogoImageUrl { get; set; }
        public string AboutDescription { get; set; }
        public string AboutPictureUrl { get; set; }
        public string favIconUrl { get; set; }
        public string FacebookLink { get; set; }
        public string Instagram { get; set; }
        public string Pinterest { get; set; }
        public string LinkedIn { get; set; }
        public string Twitter { get; set; }
        public string ContactEmail { get; set; }
        public string ContactNumber { get; set; }
    }
}

File: client/src/app/shared/models/appDetails.ts
import { ISelectedFile } from "../components/file-upload/file-upload.component";

export interface IAppDetails {
    companyName?: string;
    mainLogoImageUrl?: string;
    aboutDescription?: string;
    aboutPictureUrl?: string;
    contactPictureUrl?: string;
    favIconUrl?: string;
    facebookLink?: string;
    instagram?: string;
    pinterest?: string;
    linkedIn?: string;
    twitter?: string;
    contactEmail?: string;
    contactNumber?: string;
    id: number;
}

export interface IAppDetailsView {
    companyName: string;
    mainLogoImage?: ISelectedFile;
    mainLogoImageUrl?: string;
    aboutDescription?: string;
    aboutPicture?: ISelectedFile;
    aboutPictUrl?: string;
    contactPicture?: ISelectedFile;
    contactPicUrl: string;
    favIconUrl?: string;
    facebookLink?: string;
    instagram?: string;
    pinterest?: string;
    linkedIn?: string;
    twitter?: string;
    contactEmail?: string;
    contactNumber?: string;
    id: number;
}
 